src/test/java/it/gov/pagopa/payment/dto/mapper/barcode/TransactionBarCodeCreationRequest2TransactionInProgressMapperTest.java

```diff
@@ -0,0 +1,54 @@
+package it.gov.pagopa.payment.dto.mapper.barcode;
+
+import it.gov.pagopa.payment.constants.PaymentConstants;
+import it.gov.pagopa.payment.dto.barcode.TransactionBarCodeCreationRequest;
+import it.gov.pagopa.payment.dto.mapper.TransactionBarCodeCreationRequest2TransactionInProgressMapper;
+import it.gov.pagopa.payment.enums.OperationType;
+import it.gov.pagopa.payment.enums.SyncTrxStatus;
+import it.gov.pagopa.payment.model.TransactionInProgress;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.time.LocalDateTime;
+import java.time.OffsetDateTime;
+
+class TransactionBarCodeCreationRequest2TransactionInProgressMapperTest {
+    private TransactionBarCodeCreationRequest2TransactionInProgressMapper mapper;
+
+    @BeforeEach
+    void setUp() {
+        mapper = new TransactionBarCodeCreationRequest2TransactionInProgressMapper();
+    }
+
+    @Test
+    void applyTest() {
+
+        TransactionBarCodeCreationRequest trxCreationReq = TransactionBarCodeCreationRequest.builder()
+                .initiativeId("INITIATIVEID")
+                .build();
+        OffsetDateTime now = OffsetDateTime.now();
+        TransactionInProgress result =
+                mapper.apply(
+                        trxCreationReq, "CHANNEL", "USERID");
+
+        assertResponse(trxCreationReq, now, result);
```

Comment:
check if there are not unexpected null fields (it.gov.pagopa.common.utils.TestUtils#checkNotNullFields)